wrappers/node/src/rustlib.ts

```diff
@@ -427,7 +427,7 @@ export const FFIConfiguration: { [ Key in keyof IFFIEntryPoint ]: any } = {
   vcx_set_logger: [FFI_ERROR_CODE, [FFI_VOID_POINTER, FFI_CALLBACK_PTR, FFI_CALLBACK_PTR, FFI_CALLBACK_PTR]],
 
   // mock
-  vcx_set_next_agency_response: [FFI_VOID, [FFI_UNSIGNED_INT]],
+  vcx_set_next_agency_response: [FFI_VOID, [FFI_STRING]],
```

Comment:
Maybe we should remove this method?

Reply:
Probably yes, as we've discussed, this is not wrapper testing approach we want to take. Will remove in separate PR., For now fixing the FFI error I've introduced.